COLLABORATOR_GUIDE.md

```diff
@@ -416,14 +416,15 @@ longer be used.
 
 Node.js uses three Deprecation levels:
 
-* *Documentation-Only Deprecation* refers to elements of the Public API that are
-  being staged for deprecation in a future Node.js major release. An explicit
-  notice indicating the deprecated status is added to the API documentation
-  but no functional changes are implemented in the code. There will be no
-  runtime deprecation warnings emitted for such deprecations by default.
-  Documentation-only deprecations may trigger a runtime warning when Node.js
-  is started with the [`--pending-deprecation`][] flag or the
-  `NODE_PENDING_DEPRECATION=1` environment variable is set.
+* *Documentation-Only Deprecation* refers to elements of the Public API that
+  should be avoided by developers and that might be staged for deprecation in a
```

Comment:
The word "deprecation" does not mean removal. It means discouragement of use. So "staged for deprecation" makes no sense in this context. A documentation deprecation *is* a deprecation. I think you might mean "staged for removal"? Although I would just go with "removed".

Reply:
I see that "staged for deprecation" was existing text, not new text introduced here. I'd still encourage removing it as it is ambiguous at best. "Removed" is clear. Or if you mean a runtime warning, then say that. Either way, more precise and explicit is better.

Reply:
(Above comments are suggestions, but not blocking objections.)